design/video-infrastructure.md

```diff
@@ -2,49 +2,49 @@
+We can also decrease the image quality for viewers with slower connections or smaller displays. If the viewer has a slower connection, we still want to provide them with a stream to watch and to do this. We need to be able to provide a lower-quality stream for them. Or if they have a smaller display providing a higher resolution stream does not make any difference and can sometimes look worse.
 
 ### Video Codecs
 
-There are a few video codes mentioned in the above paragraph. Generally H264 has been the golden standard for most if not all of live video for a very long time, due to the large amount of client support. However AV1 provides much better quality per bit compression. On average it results in about 50% smaller files. So a 8k bitrate in H264 would be a 4k bitrate stream in AV1. This allows us to ship higher quality video to viewers with slower internet speeds and also allows us to save a lot of money in CDN caching and bandwidth costs. However AV1 is much harder to transcode than H264. So transcoding becomes a higher cost.  
+There are a few video codes mentioned in the above paragraph. Generally, H264 has been the golden standard for most, if not all, of live video for a very long time due to a large amount of client support. However, AV1 provides much better quality per-bit compression. On average, it results in about 50% smaller files. So an 8k bitrate in H264 would be a 4k bitrate stream in AV1. This allows us to ship higher-quality video to viewers with slower internet speeds and also allows us to save a lot of money in CDN caching and bandwidth costs. However, AV1 is much harder to transcode than H264. So transcoding becomes a higher cost.
 
 ## Edge
 
-We are going to have many servers in multiple regions around the world like Europe, North America, South America, South East Asia, Australia, Africa.
+We will have many servers in multiple regions worldwide, like Europe, North America, South America, South East Asia, Australia, and Africa.
 
 These servers will cache the video locally so that viewers from those regions can get the lowest latency to broadcast possible.
 
-Caching the video locally also means that viewers can also watch at a higher quality since the bandwidth will be local and is likely to be faster & cheaper than international bandwidth.
+Caching the video locally also means that viewers can watch at a higher quality since the bandwidth will be local and is likely to be faster & cheaper than international bandwidth.
 
-Edge is a very complex system designed to do these functions efficiently, reliably and cost effectively. So for that reason there is a [seperate flowchart diagram for edge](./cdn-edge.md).
+Edge is a complex system designed to do these functions efficiently, reliably, and cost-effectively. So for that reason, there is a [separate flowchart diagram for edge](./cdn-edge.md).
```

Comment:
"is a rather complex system" could sound cooler. And in the last part "edge' is not capitalized.

Reply:
Seconded